frontend/src/pages/projects/screens/detail/notebooks/useNotebookImageData.ts

```diff
@@ -196,8 +250,30 @@ export const getImageAvailability = (imageStream: ImageStreamKind): NotebookImag
 export const getDeletedImageData = (
   imageDisplayName: string | undefined,
 ): NotebookImageData[0] => ({
-  imageAvailability: NotebookImageAvailability.DELETED,
+  imageStatus: NotebookImageStatus.DELETED,
   imageDisplayName,
 });
 
+const getImageStatus = (
+  imageVersion: ImageStreamSpecTagType,
+  imageStatus: NotebookImageAvailability | NotebookImageStatus,
+):
```

Comment:
get image status should just return the status. right now it is returning both status or availability